sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/ControlBlockNetworkSettings.java

```diff
@@ -24,10 +24,10 @@ public interface ControlBlockNetworkSettings {
      * vlanPriority will be ignored when vlanId is null.
      *
      * @param controlBlockAdapter ControlBlock for which we want to configure the communication section
-     * @return network settings : All fields are optional (meaning fields can be null).
-     * When the return value itself is null, the communication section will not be configured for this ControlBlock.
+     * @return network settings to use to configure Communication section for this ControlBlock.
+     * An error message can be provided (i.e. errorMessage not null) or a null settings, in order not to configure the ControlBlock.
```

Comment:
```suggestion
     * An error message can be provided (i.e. errorMessage not null) or a null settings, in order to not configure the ControlBlock.
```

Reply:
I put "in order to avoid configuring the ControlBlock" to have an affirmative sentence.